README.md

```diff
@@ -1,3 +1,5 @@
+![build status](https://circleci.com/gh/docker/cli.svg?style=shield&circle-token=3de88f141d4323ea4971a870ef706abc6740ef27)
```

Comment:
Oh, you are right. Thank you.
It is a RO token having only access to build status. I am destroying it anyway.

A link to the build would be nice IMHO...